spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -108,6 +111,8 @@
 
     private static final String GITHUB_API_EMAIL = GITHUB_API_USER + "/emails";
 
+    private static final String GATEWAY_HEALTH_MONITOR_FRAGMENT = "admin/fragments/gateway_health_monitor :: gateway-health-monitor";
```

Comment:
I don't know how the details, but is it necessary to hard-code this dependency? I would have thought, that the communication between front-end and back-end could be handled indirectly by, e.g., the ViewController.

Reply:
So this is a fragment, not a dependency.
This fragment is used to populate the Gateway Health Monitor with data, without the need of long functions to structure it by hand the whole code on Javascript and HTML. So as this is a specific fragment, I need to tell Spring where it is and which fragment name

Reply:
Ah okay. Thanks for the explanation. I have not seen a class like that before but it makes sense to me now.

Reply:
No worries, you told that you wanted to learn more about the fragments, this is a good example :)